packages/storybook/stories/Components/Charts/LineChart.stories.tsx

```diff
@@ -6,7 +6,6 @@ const { defaultExport, createStory } = createComponentStories({
   component: LineChart,
   args: {
     height: 300,
-    categories: ["uv", "pv", "amt"],
```

Comment:
minor: no need to remove it from here. You can override it just in the last story